src/style.css

```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
+
+html {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
bad idea to reset margin/padding for almost all elements on page
please reset it exactly for elements you need